contracts/token/ERC721/ItemNFT.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-FileCopyrightText: 2023 ISKRA Pte. Ltd.
+// SPDX-License-Identifier: MIT
+// @author iskra.world dev team
+
+pragma solidity ^0.8.2;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/access/Ownable2Step.sol";
+
+// ItemNFT features
+//  - ERC721 basic functions
+contract ItemNFT is ERC721, Ownable2Step {
```

Comment:
discussion: burnning
For the case of Mystery Box's reveal, it needs to be determined how should be done for the collected NFT.
- Add a burn function and grant permission to the reveal contract
- Send them to the account like 0xdead.
- Simply keep them

Reply:
I think we should apply MultiToken's burn policy equally to ItemNFT. However, a separate PR is needed.